modules/nf-core/propr/propd/main.nf

```diff
@@ -10,17 +10,21 @@ process PROPR_PROPD {
     input:
     tuple val(meta), path(count)
     tuple val(meta2), path(samplesheet)
+    val GSEA
 
     output:
     tuple val(meta), path("*.propd.rds"), emit: propd
     tuple val(meta), path("*.propd.tsv"), emit: results
     tuple val(meta), path("*.fdr.tsv")  , emit: fdr         , optional:true
+    tuple val(meta), path("*.gct")  , emit: gct         , optional:true
```

Comment:
For readability, you have to have the different elements of different outputs at same columns.
If you check the emit in session_info and versions, it follows a lot of spaces to be in the same place as the emits for propd, results and fdr. The same for optional.

Reply:
Sorry I should have specify it in the issue. There is a CUSTOM_TABULARTOGSEACLS module already implemented in nf-core. So we don't need to produce cls file from propr module.
What it is still nice that the GCT file you produce will be based on the ranked differential proportionality values/ p-values /etc.

Reply:
I have aligned the commas. Regarding the CLS files, I was not aware of the existing module, so I will just remove the writeCLS functions from the template